### DISCUSSION TITLE
Welcome to SuperNOVAS Discussions!

### DISCUSSION THREAD

**Discussion by attipaci** [Sigmyne/SuperNOVAS#2] [Announcements]
## 👋 Welcome!
  Discussions is a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by giapiccoli** [comment]
Good day, I am trying to use SuperNovas on Debian, I did the installation as suggested, I have also tried to modify config.mk according to my needs but when trying to compile example.c I always get the following error, even trying different configurations for config.mk:

_/progs/snovas$ gcc example.c -o example -lnovas -lm
/usr/bin/ld: /tmp/ccgIWuDI.o: warning: relocation against `T0' in read-only section `.text'
/usr/bin/ld: /tmp/ccgIWuDI.o: in function `main':
example.c:(.text+0x96a): undefined reference to `T0'
/usr/bin/ld: warning: creating DT_TEXTREL in a PIE
collect2: error: ld returned 1 exit status_

Can someone help me ?

Thank you,
Gianni Piccoli

**Reply by attipaci** [reply]
Ciao @giapiccoli,

Yes, that was an oversight of mine that is discussed in PR #17, and the fix has been committed to the `v1.0.1` development branch. So, just switch to that branch in you local repo, and you should be good to go. Else you can also grab the tarball for the [v1.0.1-rc1](https://github.com/Smithsonian/SuperNOVAS/releases/tag/v1.0.1-rc1) under Releases.

Sorry for the mishap. The next release (around June 1), based on that branch, will also contain the fix.

**Comment by giapiccoli** [comment]
Ok, thank you very much for the quick answer and the great work.

Regards, Gianni Piccoli

**Reply by attipaci** [reply]
You are very welcome! 

As a side comment, I did not really care much about that example -- which is included mostly only because it was part of the original NOVAS distribution. I think the README provides much a much better cookbook, with a better approach than the example. The example is a bit dated way of doing things...

**Comment by giapiccoli** [comment]
Thank you, in fact I was recreating an example using the readme.
Regards.

**Reply by attipaci** [reply]
Well, the `T0` constant (which has been deprecated) is now available as `NOVAS_JD_J2000` by default. Else you can include `novascon.h` for a declaration and then link with `novascon.c` for the old (not so good) ways of NOVAS... I hope you got it all sorted out.
